pimd/pim_ifchannel.c

```diff
@@ -707,6 +707,21 @@ static int on_ifjoin_prune_pending_timer(struct thread *t)
 
 				pim_jp_agg_single_upstream_send(&parent->rpf,
 								parent, true);
+				/*
+				 * SGRpt prune pending expiry has to install to
+				 * drop the SG traffic with none oil incase no
```

Comment:
Sorry -- I'm struggling to parse this sentence. :-) I think you mean "SGRpt prune has to be installed until it expires to drop the SG traffic in case no other interface exists"? The OIL should be empty in this case, I think, which is what the "none oil" phrase means? Just want to make certain other readers in the future will understand what's going on with this bit of code. :-)

Reply:
@riw777 Thanks for reviewing.
You are right in what I meant. Let me change the 'none oil' to empty olist.

Reply:
@riw777 changed the sentence a bit. Can you please check and let me know?